response.ts

```diff
@@ -22,6 +22,18 @@ function isReader(value: any): value is Deno.Reader {
     typeof value.read === "function";
 }
 
+const ENCODE_CHARS_REGEXP = /(?:[^\x21\x25\x26-\x3B\x3D\x3F-\x5B\x5D\x5F\x61-\x7A\x7E]|%(?:[^0-9A-Fa-f]|[0-9A-Fa-f][^0-9A-Fa-f]|$))+/g
```

Comment:
Can we move this whole bit to `/util.ts`.  It makes more sense there.

Reply:
Did it. Not sure about where to place the constants. But if you think It's not OK, please tell me.